index.js

```diff
@@ -442,12 +463,24 @@ app.post('/api/heartbeat', Validator.body(['coreid', 'event', 'data']).exists(),
         response.status(200).json(errorMessage)
       } else {
         const message = JSON.parse(request.body.data)
-        const doorStatus = message.door_status
-        const doorTime = message.door_time
-        const insTime = message.ins_time
-
-        await redis.addEdgeDeviceHeartbeat(location.locationid, doorStatus, doorTime, insTime)
-
+        const missedDoorMessagesCount = message.doorMissedMsg
+        const doorLowBatteryFlag = message.doorLowBatt
+        const millisSinceDoorHeartbeat = message.doorLastHeartbeat
+        const resetReason = message.resetReason
+        const stateTransitionsArray = message.states.map(convertStateArrayToObject)
+        if (doorLowBatteryFlag && !lowBatteryAlertTimeout(location.sentLowBatteryAlertAt)) {
+          helpers.logSentry(`Received a low battery alert for ${location.locationid}`)
+          sendSingleAlert(location.locationid, `The battery for the ${location.displayName} door sensor is low, and needs replacing.`)
+          await db.updateLowBatteryAlertTime(location.locationid)
+        }
```

Comment:
Is this the exact same code as in `/api/door`? If so, it would nice to have it factored out into a function to make sure that we are consistent in the two places.